patches/@eslint+eslintrc+0.4.0.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/@eslint/eslintrc/lib/config-array-factory.js b/node_modules/@eslint/eslintrc/lib/config-array-factory.js
+index c7ff6a0..6a88c6d 100644
+--- a/node_modules/@eslint/eslintrc/lib/config-array-factory.js
++++ b/node_modules/@eslint/eslintrc/lib/config-array-factory.js
+@@ -41,7 +41,6 @@
+ 
+ const fs = require("fs");
+ const path = require("path");
+-const importFresh = require("import-fresh");
+ const stripComments = require("strip-json-comments");
+ const ConfigValidator = require("./shared/config-validator");
+ const naming = require("./shared/naming");
+@@ -222,7 +221,7 @@ function loadLegacyConfigFile(filePath) {
+ function loadJSConfigFile(filePath) {
+     debug(`Loading JS config file: ${filePath}`);
+     try {
+-        return importFresh(filePath);
```

Comment:
`import-fresh` is a Sindre Sorhus package with Sindre Sorhus dependencies with too-clever-by-half Sindre Sorhus hacks that are fundamentally incompatible with LavaMoat. 

`import-fresh` is basically native `require`/`import`, but without retaining any module state created by previous imports of the module. I don't know why the ESLint authors wanted to use it here, but it seems to work just fine without it.